nextflow_schema.json

```diff
@@ -654,7 +666,7 @@
                 },
                 "miso_genes": {
                     "type": "string",
-                    "default": "ENSG00000004961, ENSG00000005302",
+                    "default": "ENSG00000004961, ENSG00000005302, ENSG00000147403",
                     "description": "List containing identifiers of genes to plot."
```

Comment:
please specify the format of the list, and add a pattern entry to avoid csv/tsv confusion etc.

Reply:
can I get an example of the pattern?

Reply:
something like
```suggestion
                    "description": "List containing identifiers of genes to plot."
                    "pattern": "^[\\w-]+(,\\s+[\\w-]+)*$"
```